### PR TITLE
Added support for le_chiffre_stm32

### PR DESCRIPTION
**Pull request by BeatGrate**

### CODE REVIEW

File: .github/workflows/outboards/boards/le_chiffre_stm32 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Copyright 2021 Manna Harbour
```
Comment (manna-harbour) [inline .github/workflows/outboards/boards/le_chiffre_stm32:1]: Please update the dates in any new or modified files.

**Comment by manna-harbour** [comment]
Thanks for the PR!

**Comment by manna-harbour** [comment]
Thanks!

**Comment by BeatGrate** [comment]
Thank you!!

**Pull request merged**